app/helpers/urlUtils.ts

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const isHttpOrHttps = (url?: string) => {
+  if (!url) {
+    return false
+  }
+  return /^http(?=s:|:)/.test(url)
```

Comment:
Slightly easier to read way to match here:
`/^https?:/`

Reply:
Actually `/^https?:/i` so you can test with `HTTPS://`. Please add a test case for that one too.

Reply:
done 👍